content/examples/01-next-app/01-basics/02-streaming-text-generation.mdx

```diff
@@ -58,8 +58,7 @@ import { streamText } from 'ai';
 import { openai } from '@ai-sdk/openai';
 import { createStreamableValue } from 'ai/rsc';
 
-export async function generate(input: string) {
-  'use server';
```

Comment:
Why was `use server` removed here?

Reply:
since we had  ```use server``` at the top of the file there was no need for it ..or am i wrong

Reply:
ah yes